prestapyt/prestapyt.py

```diff
@@ -294,7 +294,7 @@ def _validate_query_options(self, options):
             )
         supported = (
             'filter', 'display', 'sort',
-            'limit', 'schema', 'date', 'id_shop'
+            'limit', 'schema', 'date', 'id_shop', 'id_group_shop',
```

Comment:
Isn't `id_shop` already for multi-shop?

Reply:
I'm agree with this, but this requires that you create one call for each shop, even if the shops are in the same group.
The `id_group_shop` option allows you to update data for each shops in the same group with one call.

Reply:
I don't figure out right now how to update multiple data from different shops...

Reply:
It's not for requests Prestashop -> Odoo but for requests Odoo -> Prestashop such as product updates with `connector_prestashop_catalog_manager` Odoo module.

It's simply an update to match with the PHP Library : https://github.com/PrestaShop/PrestaShop-webservice-lib/blob/master/PSWebServiceLibrary.php#L341